src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -249,6 +264,12 @@ const Jazz: React.FC<JazzProps> = ({ setUserList, venue }) => {
                     />
                   )} */}
                 </div>
+              ) : (
+                <TablesControlBar
+                  className="control-bar"
+                  handleChecked={onToggleTables}
+                  isChecked={isChecked}
+                />
```

Comment:
`!shouldShowReactions` is not equal to `!seatedAtTable` which should be the right thing to check. As far as I understand reactions can be turned off in the venue settings as well and in this case you'll see `TablesControlBar`, which is not correct. 
so, please, fix that.

Reply:
fixed in e8298bec7e35a4e40e7175b130015db2080f52e5